client/my-sites/customer-home/cards/tasks/site-setup-list/nav-item.jsx

```diff
@@ -43,7 +43,12 @@ const NavItem = ( { text, taskId, isCompleted, isCurrent, onClick, showChevron }
 			<div className="nav-item__text">
 				<h6>{ text }</h6>
 			</div>
-			{ showChevron && <Gridicon className="nav-item__chevron" icon="chevron-right" size={ 18 } /> }
+			{ useDrillLayout &&
+				( isCurrent ? (
+					<Gridicon className="nav-item__chevron" icon="chevron-up" size={ 18 } />
```

Comment:
Same here with the simpler ternary.

Reply:
Oh thanks, I actually forgot to refactor this component.